ajax/libs/sinon.js/package.json

```diff
@@ -16,6 +16,15 @@
     "type": "git",
     "url": "https://github.com/sinonjs/sinon"
   },
+  "npmName": "sinon",
+  "npmFileMap": [
+    {
+      "basePath": "pkg",
+      "files": [
+        "**/+(sinon.|sinon-no-sourcemaps.)*"
```

Comment:
@sufuf3 Sorry...I can't really understand what are you talking about 
>First, is using auto-update but the basePath should be "" (but why can't we use pkg, if there are no dist or build and it seem that the OK files always in pkg folder?)

but I guess you want to discuss "can pkg as basePath"? Actually I can't be sure of can we or can we not use it as basePath. So I used "?" to express that it's a question.
For another question. Yes, we can get the files in the old versions with auto-update but the current pattern can't get everything we want. For example, sinon-ie.js. You can take a look at https://cdnjs.com/libraries/sinon.js.